functions/utils.js

```diff
@@ -85,10 +90,14 @@ export async function getZip_Uid(data) {
   }
 }
 
-export function getRootUrlWithoutTrailingSlash(url) {
+/**
+ * Create a file on GitHub with Octokit.
+ * @param {string} url
+ * @returns {string} rootUrl
+ */
+export async function getRootUrlWithoutTrailingSlash(url) {
```

Comment:
why this should be an async function ?

Reply:
@vfdev-5 I tested the netlify function locally and it was giving some error so I wanted to test this here in production, if it doesn't work lets revert it